_posts/2017-10-05-apprenticeship-at-8thlight.md

```diff
@@ -0,0 +1,213 @@
+
+You get paid and you learn full-time. Their [careers site] underlines that this is an essential part of the 8th Light community.
+> While they aren't a billable member of a client project, they're still part of the 8th Light team and we believe they should be paid in that role.
+
+This is not only an investment and encouragement for the people they hire but also a long-term investment in the company. The people that are treated like this are more likely to be loyal to the company, even though they are not billable yet.
+
+[promote-apprenticeship]: https://8thlight.com/blog/paul-pagel/2017/08/14/diversity-inclusion-8th-light.html
+[careers site]: https://8thlight.com/job-opportunities/london-crafter/
+
+## Safe environment
+
+One fundamental thing for making this an encouraging, collaborative and positive experience for everyone is to provide a safe environment. There is much more than the things written here that make a safe environment, but I would like to pick out some things. 
+
+Makis, a crafter wrote on [his blog][makis-teach] that the London office started something called Zagaku, where “every morning a crafter or an apprentice will teach a topic for 20-30 minutes to the apprentices”. And this is not a one-sided learning but “learning happens on both ends (crafter & apprentices)” and by doing that internally a safe environment is created. People know each other, have a good level of communication skills and respect each other. This encourages everyone, not only the apprentices to speak up, to ask questions, to learn and to teach. 
+
+I witnessed this being the case. The office was a very collaborative environment. People were patient and calm. Many had been collaborating, for example by pairing.
+
+I think that also the hardware setup on all desks is supporting this. Because every desk was equipped with a big monitor and two keyboards that were ready to use. That means every desk was inviting the people to pair with each other. And there had been many apprentice+crafter pairs in the office. This makes pairing very easy and look like the normal thing.
+
+[makis-teach]: http://makisotman.com/to-teach-is-to-learn/
+
+## Inclusiveness
+
+I think no further explaining is needed that a safe environment is a basic thing (to have in every office). Additionally we saw that 8th Light is doing a great job in being more inclusive. The first day we had been in the office Tobias and me had been the only men in the room. When we walk on the street we expect to be surrounded by all kinds of people. In a tech company this is not the case yet, and it was feeling good to work with woman and see that is reality in this office, just as much as on the street. Jim told us that 8th Light has about 40% of women employed. Good job! I guess if 8th Lighters look into other IT companies they must feel quite strange when they realise the absence of woman and see just less diverse groups of people work there.
```

Comment:
We're not quite at 40% across the company, but we're close. We have not released any numbers, though we've begun tracking this internally. I'd prefer not to be quoted on the number if that's okay, a glance at our team pages will certainly give you a sense for where we're at.

Reply:
wont quote on numbers :)